server/src/test/java/io/crate/types/TimeTypeTest.java

```diff
@@ -0,0 +1,186 @@
+package io.crate.types;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import java.util.function.Function;
+
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertThat;
+
+public class TimeTypeTest {
+
+    @Rule
+    public ExpectedException expectedException = ExpectedException.none();
```

Comment:
maybe extends `CrateUnitTest`, you also will get  `ExpectedException` from base class